ietf/settings/base.py

```diff
@@ -173,8 +173,8 @@
 
 CACHES = {
     "default": {
-        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
-        "LOCATION": "cache_table",
+        "BACKEND": "django.core.cache.backends.memcached.PyMemcacheCache",
```

Comment:
Consider splitting configuration `session` out even if you also point `default` to memcached. 

Also take a look at datatracker's ietf.utils.cache.LenientMemcacheCache to see if it might be useful here.